db/migrate/20210313061212_remove_client_from_access_log.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveClientFromAccessLog < ActiveRecord::Migration[6.0]
+  # Follow up migration for AddRecordToAccessLog
+  def up
+    remove_reference :access_logs, :client
+    change_column_null :access_logs, :record_id, false
```

Comment:
This line might fail if any AccessLog records were creating when the old code was running. Just a FYI; not a blocker; if this is a problem, then the migration will happily undo it self due to transactions. :)

Reply:
Do you think I should do the same update to these columns that I did in the earlier migration? I imagine that would make it more likely to succeed.
(set record_type to "Client", set record_id to "client_id")